ansible/roles/delfin-installer/scenarios/source-code.yml

```diff
@@ -91,3 +91,104 @@
     option: backend_server
     value: "{{ delfin_redis_ip }}:{{ delfin_redis_port }}"
   become: yes
+
+# Performance Export Configurations
+- name: Check and remove performance exporters configs
+  ini_file:
+    create: no
+    state: absent
+    path: "/opt/delfin-linux-amd64/etc/delfin/delfin.conf"
+    section: DEFAULT
+    option: performance_exporters
+    value: ""
+  become: yes
+  when:
+    - performance_exporters == None
+
+- name: Enable Performance Exporter configuration
+  ini_file:
+    create: no
+    path: "/opt/delfin-linux-amd64/etc/delfin/delfin.conf"
+    section: DEFAULT
```

Comment:
Is there a default performance exporter set if `performance_exporters` is not set?

Reply:
yes there is an example exporter which just prints the metics in log.. example exporter is a reference to develop real exporters.